mlrun/api/main.py

```diff
@@ -345,9 +345,10 @@ async def _start_log_for_run(
             runtime_handler = await fastapi.concurrency.run_in_threadpool(
                 get_runtime_handler, run_kind
             )
+            object_id = runtime_handler.resolve_object_id(run)
```

Comment:
you forgot deleting the `run_uid` above.

Reply:
I didn't delete it on purpose, as it is used in the function for other things like for calling `start_log`.
It is also returned by the function.